news/12753.doc.rst

```diff
@@ -0,0 +1 @@
+Add example for install by git+ssh
```

Comment:
Please, use complete sentences (with a trailing period) in a non-imperative mood, as the change log is meant to describe what's changed since the previous release. So it's not a TODO-list style entry, since it's not something planned for the future at this point.

Reply:
Thanks for the review. Do you mean add a `.` at the end?

Reply:
Yes, but also make it possible for the readers to understand what's changed and how it affects them w/o looking into commits.